src/igvfd/tests/test_audit_measurement_set.py

```diff
@@ -196,6 +197,19 @@ def test_audit_readout(
         for error in res.json['audit'].get('NOT_COMPLIANT', [])
     )
 
+    # Assay term should not be equivalent to readout term
```

Comment:
i think the test should cover both conditions of the audit, assay requiring readout and assay that should not have a readout

Reply:
There are already tests above for this. I think every audit of the 3 cases is tested, not sure what else to test.